helpers/make_template.py

```diff
@@ -50,15 +53,16 @@ def main (options_file, template_file):
                     entered_restart_section=True
 
                 key = (l.split("=")[0]).strip().lower()
-                if key == "restart_file":
-                    if verbose: print("Writing restart_file line")
-                    tmpl.write('    restart_file="__RESTART_FILE__"\n')
-                elif key == "restart_date":
+                ## as long as the default (i.e. restart_file="restart/icar_rst_" ) is used, no need to change.
+                # if key == "restart_file":  
+                #     if verbose: print("Writing restart_file line")
+                #     tmpl.write('    restart_file="__RESTART_FILE__"\n')
```

Comment:
keep this at least for now as it seems like it might be needed for some workflows